pkg/services/alerting/notifiers/slack.go

```diff
@@ -73,25 +75,50 @@ func init() {
         </info-popover>
       </div>
       <div class="gf-form max-width-30">
-        <span class="gf-form-label width-6">Mention</span>
+        <span class="gf-form-label width-8">Mention Users</span>
         <input type="text"
           class="gf-form-input max-width-30"
-          ng-model="ctrl.model.settings.mention"
+          ng-model="ctrl.model.settings.mentionUsers"
           data-placement="right">
         </input>
         <info-popover mode="right-absolute">
-          Mention a user or a group using @ when notifying in a channel
+          Mention one or more users (comma separated) when notifying in a channel, by ID (you can copy this from the user's Slack profile)
         </info-popover>
       </div>
       <div class="gf-form max-width-30">
-        <span class="gf-form-label width-6">Token</span>
+        <span class="gf-form-label width-8">Mention Groups</span>
+        <input type="text"
+          class="gf-form-input max-width-30"
+          ng-model="ctrl.model.settings.mentionGroups"
+          data-placement="right">
+        </input>
+        <info-popover mode="right-absolute">
+          Mention one or more groups (comma separated) when notifying in a channel (you can copy this from the group's Slack profile URL)
+        </info-popover>
+      </div>
+      <div class="gf-form max-width-30">
+        <span class="gf-form-label width-8">Mention Channel</span>
+        <select
+          class="gf-form-input max-width-30"
+          ng-model="ctrl.model.settings.mentionChannel"
+          data-placement="right">
+		  <option value="">Disabled</option>
+		  <option value="here">Every active channel member</option>
+		  <option value="channel">Every channel member</option>
```

Comment:
there is an additional `everyone` [option](https://api.slack.com/reference/surfaces/formatting#special-mentions) that notifies every person in the #general channel (ie. every non-guest member of a workspace).

Reply:
@papagian Yeah, but I didn't really think supporting `@everyone` really made sense since this notification is for a channel. My reasoning was that if you are notifying a channel, why would you also want to notify `#general`? I could be convinced I'm wrong though, maybe there are valid use cases?

Reply:
ok, you have convinced me :)